### DISCUSSION TITLE
IP static or dynamic

### DISCUSSION THREAD

**Discussion by migueGodoy** [Labs64/NetLicensing-Community#112] [Questions] [answered]
Hi,

One of our customers is asking whether the ip behind https://go.netlicensing.io/core/v2/rest is static or dynamic so they know if they should apply one rule to their firewall or not.

Could you please let me know?

Thanks in advance

Migue Godoy
Power Electronics

**Comment by r-brown** [comment, marked as answer]
Hello @migueGodoy 

Similar question was replied here already: **Firewall: whitelist Netlicensing Services IP address(-es) #44**
